src/components/Offers/Form/form-components/OfferForm.js

```diff
@@ -0,0 +1,378 @@
+                                            name="fields"
+                                            render={(
+                                                { field: {  onBlur, name } },
+                                            ) => (
+                                                <MultiOptionAutocomplete
+                                                    name={name}
+                                                    onBlur={onBlur}
+                                                    error={errors.fields || requestErrors.fields}
+                                                    disabled={formDisabled}
+                                                    chipWrapperProps={{
+                                                        className: classes.autocompleteChipWrapper,
+                                                    }}
+                                                    textFieldProps={{
+                                                        margin: "none",
+                                                    }}
+                                                    {...fieldsSelectorProps}
+                                                    label="Fields *"
+                                                    placeholder="Fields *"
+                                                />
+                                            )}
+                                            control={control}
+                                        />
+                                    </Grid>
+                                    <Grid item xs={12} lg={6}>
+                                        <Controller
+                                            name="technologies"
+                                            render={(
+                                                { field: { onBlur, name } },
+                                            ) => (
+                                                <MultiOptionAutocomplete
+                                                    name={name}
+                                                    onBlur={onBlur}
+                                                    error={errors.technologies || requestErrors.technologies}
+                                                    disabled={formDisabled}
+                                                    chipWrapperProps={{
+                                                        className: classes.autocompleteChipWrapper,
+                                                    }}
+                                                    textFieldProps={{
+                                                        margin: "none",
+                                                    }}
+                                                    {...techSelectorProps}
+                                                    label="Technologies *"
+                                                    placeholder="Technologies *"
```

Comment:
Isn't this part of the techSelectorProps? It should be

Reply:
it appears that the search is using the `useTechSelector`, should I create two different ones?

Reply:
discussed offline, let's keep it this way